src/main/resources/langs/fr_FR.yml

```diff
@@ -199,7 +199,7 @@ administration:
     invalidNumberGiven: "Modificateur fourni invalide, le modificateur doit être un pourcentage positif entre `0,01%`et `500%`."
     resetToDefault: "Le modificateur de niveau a été réinitialisé en utilisant le modificateur par défaut **:modifier**."
     changedTo: "Le modificateur de niveau et d'expérience est à présent **:modifier**"
-    requiredIsNow: "\n\n**Required XP to reach some levels is now:**\n"
+    requiredIsNow: "\n\n**L'expérience requise pour atteindre certains niveaux est maintenant :**\n"
```

Comment:
This was (possibly) auto translated, since translations may break when using a autotranslator